src/ert/__main__.py

```diff
@@ -167,8 +167,8 @@ def valid_num_iterations(user_input: str) -> str:
 def attemp_int_conversion(val: str) -> int:
     try:
         return int(val)
-    except ValueError:
-        raise ArgumentTypeError(f"{val} is not a valid integer")
+    except ValueError as e:
```

Comment:
This looks like an automated fix. Is it given that we always want to reraise exceptions and keep all the details (it probably often is). When seen by users, more details can make them blind. 

If this is a new requirement imposed by ruff, it would like to have a global exception for it in ruff-configuration, and then we can think about solving it in a future PR.

Reply:
Not really automated, I just did not see any reason to hide the details and yes, it makes `ruff` happy.